CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Improve `decode_raw` performance ([#34])
+
```

Comment:
```suggestion

[#24]: https://github.com/alloy-rs/rlp/pull/24

```

Reply:
Resolved in c0406db0ea4dcc354a8d069b2fb986e4bc5583e6